honeybee_energy/load/lighting.py

```diff
@@ -63,6 +64,7 @@ def __init__(self, identifier, watts_per_area, schedule, return_air_fraction=0.0
         self.radiant_fraction = radiant_fraction
         self.visible_fraction = visible_fraction
         self.baseline_watts_per_area = None  # can be set by the user later
+        self._user_data = None
```

Comment:
There's no need to do this here since we are already initializing `_LoadBase`, which will set this variable to `None`. I think you'll also have to remove this for a lot of other Load classes.

Reply:
commit: loads cleaned up